test/components/views/rooms/MessageComposer-test.tsx

```diff
@@ -68,10 +77,261 @@ describe("MessageComposer", () => {
             expect(wrapper.find("MessageComposerButtons")).toHaveLength(0);
             expect(wrapper.find(".mx_MessageComposer_roomReplaced_header")).toHaveLength(1);
         });
+
+        describe("when receiving a »reply_to_event«", () => {
+            let wrapper: ReactWrapper;
+            let resizeNotifier: ResizeNotifier;
+
+            beforeEach(() => {
+                resizeNotifier = {
+                    notifyTimelineHeightChanged: jest.fn(),
+                } as unknown as ResizeNotifier;
+                wrapper = wrapAndRender({
+                    room,
+                    resizeNotifier,
+                });
+            });
+
+            it("should call notifyTimelineHeightChanged() for the same context", (done) => {
+                dis.dispatch({
+                    action: "reply_to_event",
+                    context: (wrapper.instance as unknown as MessageComposer).context,
+                });
+                wrapper.update();
+
+                setTimeout(() => {
+                    expect(resizeNotifier.notifyTimelineHeightChanged).toHaveBeenCalled();
+                    done();
+                }, 150); // higher timeout than in MessageComposer::onAction
```

Comment:
Not very nice to add all this delay to the tests, could you use fake timers and advance time instead?